machines/faceScanner.ts

```diff
@@ -198,12 +198,34 @@ export const createFaceScannerMachine = (vcImage: string) =>
           });
         },
 
-        verifyImage: context => {
+        verifyImage: async context => {
           context.cameraRef.pausePreview();
           const rxDataURI =
             /data:(?<mime>[\w/\-.]+);(?<encoding>\w+),(?<data>.*)/;
-          const matches = rxDataURI.exec(vcImage).groups;
-          return faceCompare(context.capturedImage.base64, matches.data);
+
+          let isMatchFound = false;
+          for (const vcImage of vcImages) {
```

Comment:
let's connect to understand this logic.
We are trying to verify face for all images we got, but we are breaking on first match found. If there are 2 Vcs, then face verification should happen for both VC face.

Reply:
Added this logic according to this point "Perform face verification against the image in one of the VCs" mentioned in the card swati. If we want to perform the face verification for all the VCs which has image then that might require some discussions on how we want to show the face capture screen in loop based on the number of VCs that has image.